### PR TITLE
Add support for rendering borders with radius

### PR DESCRIPTION
**Pull request by mkiael**
This is a first step in adding support for rendering borders with radius.

We are still missing support for parsing the border-radius property in our CSS parser, so we will not be able to see this in action yet.

Part #99

### CODE REVIEW

**Comment by mkiael** [comment]
I should probably add some tests since we got that setup for `render.cpp` now.

File: gfx/rect_shader.frag [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        // Otherwise, set transparent color
+        } else {
+            gl_FragColor = no_color;
+        }
+    }
+    // Otherwise, set transparent color
+    else {
+        gl_FragColor = no_color;
+    }
+}
```
Comment (robinlinden) [inline gfx/rect_shader.frag:200]: Missing newline at EOF.

**Review by robinlinden — APPROVED** [review]
Looks great! Really good job. :)
Feel free to skip adding tests for render unless you want to. You didn't touch it much, and the only test for it is my example. :P

**Review by robinlinden — APPROVED** [review]
Nice tests too!

**Pull request merged**